src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -140,20 +142,26 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       return;
     }
 
+    // Needed to get absoule path of file that configures BBCode processor
+    ClassLoader classLoader = getClass().getClassLoader();
+    File file = new File(classLoader.getResource("kefirbb.xml").getFile());
+
     // processor needed for BBCode to HTML translation
-    TextProcessor processor = BBProcessorFactory.getInstance().create();
+    TextProcessor processor = BBProcessorFactory.getInstance().create(file.getAbsolutePath());
```

Comment:
Would createFromResource() work here instead of create()?
http://static.javadoc.io/org.kefirsf/kefirbb/1.5/org/kefirsf/bb/BBProcessorFactory.html  

I think it could remove the need for some of the code above.